ts/input/tex/ams/AmsMethods.ts

```diff
@@ -46,7 +46,7 @@ let AmsMethods: Record<string, ParseMethod> = {};
 
 
 /**
- * Handle AMS array environments. 
+ * Handle AMS array environments.
```

Comment:
That's been in the previous PR already?

Reply:
It may be.  It should not cause a problem, however.